ambry-api/src/main/java/com/github/ambry/quota/QuotaChargeCallback.java

```diff
@@ -37,12 +37,12 @@
    */
   static QuotaChargeCallback buildQuotaChargeCallback(RestRequest restRequest, QuotaManager quotaManager,
       boolean shouldThrottle) {
-    RequestCostPolicy requestCostPolicy = new UserQuotaRequestCostPolicy(quotaManager.getQuotaConfig());
+    RequestQuotaCostPolicy requestQuotaCostPolicy = new SimpleRequestQuotaCostPolicy(quotaManager.getQuotaConfig());
```

Comment:
Ambry calculates request cost for two separate concerns:

The SCE cost for a user request that goes towards cost model of Ambry.
The quota charge for a user request that goes towards resource quota.
This change separates out the logic for the two concerns. Doesn't change any logic other than refactoring.